catalog/imaging/imaging-actionprovider-chip/src/main/resources/chipping.html

```diff
@@ -21,7 +21,7 @@
         <meta name="viewport" content="initial-scale=1.0, user-scalable=no, width=device-width">
 
         <script src="/chipping/jquery/1.11.0/dist/jquery.min.js"></script>
-        <link rel="stylesheet" type="text/css" href="/chipping/components-font-awesome/4.6.4/css/font-awesome.min.css">
+        <link rel="stylesheet" type="text/css" href="/chipping/components-font-awesome/4.6.3/css/font-awesome.min.css">
```

Comment:
I updated this to match the version in the pom:
https://github.com/codice/alliance/blob/8c2c2c10fa67cb423827b4e7f0c4533bda25e1e6/catalog/imaging/imaging-actionprovider-chip/pom.xml#L87-L91

Reply:
Good catch

Reply:
If we want, we could make the version a maven property and have it be filtered in the html file with something like `${compoents.font.awesome.version}`